examples/design-system-tutorial/index.html

```diff
@@ -0,0 +1,174 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8" />
+        <title>Design System Tutorial</title>
+        <link href="styles.css" rel="stylesheet" />
+    </head>
+    <body>
+        <div id="root">
+            <demo-navigation>
+                <fast-tree-view>
+                    <fast-tree-item selected>My Feed</fast-tree-item>
+                    <fast-tree-item>Microsoft</fast-tree-item>
+                    <fast-tree-item>Information Technology</fast-tree-item>
+                    <fast-tree-item>Business News</fast-tree-item>
+                    <fast-tree-item>Seattle / Tacoma</fast-tree-item>
+                    <fast-tree-item>Money</fast-tree-item>
+                    <fast-tree-item>News</fast-tree-item>
+                    <fast-tree-item>Markets</fast-tree-item>
+                </fast-tree-view>
+            </demo-navigation>
+            <div class="canvas">
+                <div class="container">
+                    <fast-demo-card>
+                        <h2 slot="heading">We're not ready for another pandemic</h2>
+                        <fast-attribution slot="attribution">
+                            The Atlantic
+                            <img
+                                src="//img-s-msn-com.akamaized.net/tenant/amp/entityid/BBY4G4r?w=16&h=16&q=100&m=6&f=png&u=t"
+                                alt="The Atlantic"
+                                slot="icon"
+                            />
+                        </fast-attribution>
+                        <img
+                            src="https://www.bing.com/th?id=ORMS.bbec3ae474a1dbbf78c519c6d136c959&pid=Wdp&w=300&h=225&qlt=60&c=1&rs=1"
+                            alt=""
```

Comment:
@nicholasrice @EisenbergEffect We can probably take this as a follow-up, but we should include alt text here.